articles/idam/idam.md

```diff
@@ -0,0 +1,83 @@
+---
+title: ID & Access Management API | UKCloud Ltd
+description: Provides information on managing permissions using the ID & Access Management API
+services: idam
+author: Simon Fish
+
+toc_rootlink: Getting Started
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: IDAM
+toc_fullpath: Getting Started/idam.md
+toc_mdlink: idam.md
+---
+
+# Identity and Access Management (IDAM)
```

Comment:
Change to match the value of the title metadata tag (my suggestion):
Understanding UKCloud's Identity and Access Management API